src/rdkafka.h

```diff
@@ -47,6 +47,7 @@
 #include <stdio.h>
 #include <inttypes.h>
 #include <sys/types.h>
+#include <netdb.h>
```

Comment:
This isn't portable, and we don't actually need anything from that file.
Instead make a forward declaration of the `struct addrinfo;` just above the doc string.

Reply:
Gotcha, done!